bootstrap/environment/variables.go

```diff
@@ -456,3 +457,15 @@ func logEnvironmentOverride(lc logger.LoggingClient, name string, key string, va
 	}
 	lc.Infof("Variables override of '%s' by environment variable: %s=%s", name, key, valueStr)
 }
+
+// GetRequestTimeout gets the configuration request timeout value from a Variables variable value (if it exists)
```

Comment:
Should this function parse the env value so callers don't have to? If , so it would return (time.Duration, error)